### PR TITLE
Address comments from Ben

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: dnsop-structured-dns-error-page.xml [hunk 1]
```diff
@@ -119,8 +118,8 @@
       website.</t>
 
       <t>This document updates the EXTRA-TEXT field of Extended DNS Error
-      [RFC8914] to provide details on the DNS filtering.  This information
-      can be parsed by the client and displayed, logged, or used for other
+      [RFC8914] to provide details on the DNS filtering. This information can
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:121]:
```suggestion
      to provide details on the DNS filtering. This information can
```

File: dnsop-structured-dns-error-page.xml [hunk 2]
```diff
@@ -119,8 +118,8 @@
...
+      be parsed by the client and displayed, logged, or used for other
       purposes.</t>
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:123]:
```suggestion
      purposes.</t>
      <t>This document updates RFC 8914.</t>
```

File: dnsop-structured-dns-error-page.xml [hunk 3]
```diff
@@ -361,10 +351,9 @@ Most importantly, EDE option does not offer
     <section anchor="option-code">
       <name>I-JSON in EXTRA-TEXT field</name>
 
-      <t>Servers compliant with this specification send I-JSON
-      data in the <xref target="RFC8914">EXTRA-TEXT field</xref> using
-      the <xref target="RFC7493">Internet JSON (I-JSON) message
-      format</xref>.</t>
+      <t>Servers compliant with this specification send I-JSON data in the
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:354]:
```suggestion
      <t>Servers that are compliant with this specification send I-JSON data in the
```

File: dnsop-structured-dns-error-page.xml [hunk 4]
```diff
@@ -376,17 +365,25 @@ Most importantly, EDE option does not offer
...
+        mis-classified DNS filtering. This field is structured as an array of
+        contact URIs (e.g., tel, sips, https). At least one contact URI MUST
+        be included. This field is mandatory.</dd>
 
         <dt>j: (justification)</dt>
-        <dd>the textual justification for this particular DNS filtering. This
-        field is mandatory.</dd>
+
+        <dd>the textual justification for this particular DNS filtering. The
+        field should be treated only as diagnostic information for IT staff.
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:377]:
```suggestion
        field SHOULD be treated only as diagnostic information for IT staff.
```

File: dnsop-structured-dns-error-page.xml [hunk 5]
```diff
@@ -443,13 +440,13 @@ Most importantly, EDE option does not offer
...
 
         <ul>
-          <li>The response MUST be received over an
-          encrypted DNS channel. If not, the requestor MUST discard data in
-          the EXTRA-TEXT field.</li>
+          <li>The response MUST be received over an encrypted DNS channel. If
+          not, the requestor MUST discard data in the EXTRA-TEXT field.</li>
+
+          <li>The response MUST be received from a DNS server which advertised
+          EDE support via <xref
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:447]:
```suggestion
          EDE support via a trusted channel, e.g., <xref
```

File: dnsop-structured-dns-error-page.xml [hunk 6]
```diff
@@ -499,8 +498,8 @@ Most importantly, EDE option does not offer
           client is implementation dependent <xref target="RFC5625" />.
           Implementations MAY choose to not forward the JSON information, or
           they MAY choose to create a new EDE option that conveys the
-          information in the "c" and "j" fields encoded in the JSON object.
-          </li>
+          information in the "c", "s" and "j" fields encoded in the JSON
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:501]:
```suggestion
          information in the "c", "s", and "j" fields encoded in the JSON
```

File: dnsop-structured-dns-error-page.xml [hunk 7]
```diff
@@ -536,7 +536,7 @@ Most importantly, EDE option does not offer
   ============== NOTE: '\' line wrapping per RFC 8792 ===============
 
   {"c":["tel:+358-555-1234567","sips:bob@bobphone.example.com", \
-  "https://ticket.example.com?d=example.org&t=1650560748"], \
+  "https://ticket.example.com?d=example.org&t=1650560748"],"s": 1, \
```
Comment (boucadair) [inline dnsop-structured-dns-error-page.xml:539]:
```suggestion
  "https://ticket.example.com?d=example.org&t=1650560748"],"s":1, \
```

**Review by boucadair — CHANGES_REQUESTED** [review]

File: dnsop-structured-dns-error-page.xml [hunk 8]
```diff
@@ -657,9 +656,138 @@ Most importantly, EDE option does not offer
...
+
+            <th align="left">Change Controller</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr>
+            <td align="left">0</td>
+
+            <td align="left">No error</td>
```
[the remark this answers is not in the document]
Comment (tireddy2) [reply dnsop-structured-dns-error-page.xml:720]: Updated to "Reserved" for better clarity.

**Pull request merged**